features/main/crud.feature

```diff
@@ -92,223 +92,229 @@ Feature: Create-Retrieve-Update-Delete
+                  {
+                      "@type": "IriTemplateMapping",
+                      "variable": "dummyFloat[gte]",
+                      "property": "dummyFloat",
+                      "required": false
+                  },
+                  {
+                      "@type": "IriTemplateMapping",
+                      "variable": "dummyFloat[lt]",
+                      "property": "dummyFloat",
+                      "required": false
+                  },
+                  {
+                      "@type": "IriTemplateMapping",
+                      "variable": "dummyFloat[lte]",
+                      "property": "dummyFloat",
+                      "required": false
+                  },
+                  {
+                      "@type": "IriTemplateMapping",
+                      "variable": "dummyPrice[between]",
+                      "property": "dummyPrice",
+                      "required": false
+                  },
+                  {
+                      "@type": "IriTemplateMapping",
+                      "variable": "dummyPrice[gt]",
+                      "property": "dummyPrice",
+                      "required": false
+                  },
+                  {
+                      "@type": "IriTemplateMapping",
+                      "variable": "dummyPrice[gte]",
+                      "property": "dummyPrice",
+                      "required": false
+                  },
+                  {
+                      "@type": "IriTemplateMapping",
+                      "variable": "dummyPrice[lt]",
+                      "property": "dummyPrice",
+                      "required": false
+                  },
+                  {
+                      "@type": "IriTemplateMapping",
+                      "variable": "dummyPrice[lte]",
+                      "property": "dummyPrice",
+                      "required": false
+                  },
+                  {
+                      "@type": "IriTemplateMapping",
+                      "variable": "dummyBoolean",
+                      "property": "dummyBoolean",
+                      "required": false
+                  },
+                  {
+                      "@type": "IriTemplateMapping",
+                      "variable": "relatedDummy.embeddedDummy.dummyBoolean",
+                      "property": "relatedDummy.embeddedDummy.dummyBoolean",
+                      "required": false
+                  },
+                  {
+                      "@type": "IriTemplateMapping",
+                      "variable": "dummyFloat",
+                      "property": "dummyFloat",
+                      "required": false
+                  },
+                  {
+                      "@type": "IriTemplateMapping",
+                      "variable": "dummyPrice",
+                      "property": "dummyPrice",
+                      "required": false
+                  }
+              ]
           }
-        ]
       }
-    }
```

Comment:
Same indentation looks weird here from github POV. This is the maintain mess I'm talking above :p.